src/main/java/com/codeborne/selenide/Condition.java

```diff
@@ -328,6 +329,11 @@ public static Condition match(String description, Predicate<WebElement> predicat
     return new CustomMatch(description, predicate);
   }
 
+  /**
+   * Check if image is loaded.
+   */
+  public static final Condition isImage = new IsImage();
```

Comment:
let's rename it to `image`.

* This is not readable: `$.should(isImage);`
* This is readable: `$.shouldBe(image);`

Reply:
done